dav2rem.py

```diff
@@ -73,11 +72,11 @@ def main():
     rdict = {splitext(basename(event.canonical_url))[0].replace('%40', '@'): event for event in calendar.events()}
 
     if args.delete:
-        local = ldict - rdict.viewkeys()
+        local = ldict - rdict.items()
```

Comment:
I guess you want `keys()` and not `items()` here (and below).

Reply:
Yes, now it works fine!